druid-shell/src/window.rs

```diff
@@ -224,6 +226,13 @@ impl WindowHandle {
         self.0.get_size()
     }
 
+    /// Set the position and/or size of the native platform window that this window manages.
+    /// This is only valid if the window is a child window. Do not use this to move a top-level
+    /// window `Window<T>`.
+    pub fn set_native_layout(&self, position: Option<Point>, size: Option<Size>) {
```

Comment:
Do we envisage these being separately set? ie why not a Rect

Reply:
That was the idea originally yes, because position and size have different semantic when it comes to repainting: position might just move the window without invalidating its content, size almost surely _will_ invalidate the content.

But as I remember it turned out that by the time the PR was opened 1. the native position was trickling down via the `PaintCtx` and leaking into the widget implementation of `paint()`, instead of being handled purely internally, and 2. the widget itself probably had little knowledge of when is best to set the size and when not. So the PR version ended up setting both at once always. I think this is more or less what happened here.

We can probably use a `Rect` now.